### PR TITLE
Wrap all comments at uniform length

### PR DESCRIPTION
**Pull request by matthchr**

### CODE REVIEW

**Review by theunrepentantgeek — APPROVED** [review]
Generally looks good. 

Would be worth double checking the places that used to use a width of 200 as that was used for "don't wrap" - things like URLs and some comment-directives for controller gen and so on.

Maybe introduce `AddUnwrappedComment()`?

File: hack/crossplane/apis/sql/v1alpha1api20201101preview/server_types_gen.go [hunk 1]
```diff
@@ -14,7 +14,8 @@ import (
 // +kubebuilder:object:root=true
 // +kubebuilder:subresource:status
 // +kubebuilder:storageversion
-//Generated from: https://schema.management.azure.com/schemas/2020-11-01-preview/Microsoft.Sql.json#/resourceDefinitions/servers
+//Generated from:
+//https://schema.management.azure.com/schemas/2020-11-01-preview/Microsoft.Sql.json#/resourceDefinitions/servers
```
[the remark this answers is not in the document]
Comment (matthchr) [reply hack/crossplane/apis/sql/v1alpha1api20201101preview/server_types_gen.go:18]: I've gone ahead and added `AddUnwrappedComment` and used that in most of the places we were using 200. I do think we may end up revisiting that as when we source from the Swagger directly, we're going to have access to the type comments as well, at which point we probably ought to include them in addition to the property comments, and we'll want to wrap the type comments so they aren't out at 200 characters.

We can cross that bridge when we get there though.

[the remark this answers is not in the document]
Comment (theunrepentantgeek) [reply hack/crossplane/apis/sql/v1alpha1api20201101preview/server_types_gen.go:18]: Pretty sure that we can both wrap _comments_ while not-wrapping other stuff (such as source URLs and directives).

**Pull request merged**